src/init/autoApplyCustomizations.ts

```diff
@@ -6,7 +6,7 @@ export function manageAutoApplyCustomizations(
   userConfig: IVSIcons,
   applyCustomizationCommand: () => void): void {
   if (!isNewVersion) { return; }
-  const propObj = packageJson.contributes.configuration.properties || {};
+  const propObj: any = packageJson.contributes.configuration.properties as object;
```

Comment:
I now got the error when using `Object` (I misread your statement). But the point still stands. I mean, what you have in this line is completely equal to `const propObj = packageJson.contributes.configuration.properties`, without any annotation at all.